app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -2,14 +2,25 @@
 
 module ExceptionHunter
   class ErrorsController < ApplicationController
+    include Pagy::Backend
+
     def index
       @errors = error_groups
     end
 
+    def show
+      Pagy::VARS[:items] = 1
+      @pagy, @error = pagy(last_error)
+    end
+
     private
 
     def error_groups
       ErrorGroup.all
     end
+
+    def last_error
+      ErrorGroup.find(params[:id]).grouped_errors.order('created_at DESC')
```

Comment:
This should be a scope and using symbols

```suggestion
      ErrorGroup.find(params[:id]).grouped_errors.ordered_by_created_at
```